resources/zib/zib-DispenseRequest.xml

```diff
@@ -0,0 +1,246 @@
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvicer-Organization" />
+      </type>
+      <mapping>
+        <identity value="zib-DispenseRequest-v1.0.3-2020EN" />
+        <map value="NL-CM:9.10.19966" />
+        <comment value="IntendedSupplier" />
+      </mapping>
+    </element>
+    <element id="MedicationRequest.note.text">
+      <path value="MedicationRequest.note.text" />
+      <short value="Comment" />
+      <definition value="Explanation for the dispense request. This explanation can contain e.g. information on why a prescriber submits a dispense request that deviates from the norm, e.g. an extra dispense request needed because the patient has lost the medication." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-DispenseRequest-v1.0.3-2020EN" />
+        <map value="NL-CM:9.10.22274" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="MedicationRequest.dispenseRequest.extension">
+      <path value="MedicationRequest.dispenseRequest.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationRequest.dispenseRequest.extension:dispenseLocation">
+      <path value="MedicationRequest.dispenseRequest.extension" />
+      <sliceName value="dispenseLocation" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-DispenseRequest.DispenseLocation" />
+      </type>
+    </element>
+    <element id="MedicationRequest.dispenseRequest.extension:validityPeriodDuration">
+      <path value="MedicationRequest.dispenseRequest.extension" />
+      <sliceName value="validityPeriodDuration" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse-Duration" />
+      </type>
+    </element>
+    <element id="MedicationRequest.dispenseRequest.validityPeriod">
+      <path value="MedicationRequest.dispenseRequest.validityPeriod" />
+      <short value="PeriodOfUse" />
+      <definition value="During the approved period of use, the pharmacist has permission to dispense medicine so that the patient has a sufficient amount of medication. In many cases, the approved period of use can be described by only an end date: the approved end date of use." />
+      <comment value="The duration of the PeriodOfUse is captured using the http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse-Duration extension." />

```

Comment:
I think it'd be a good idea to define a standard text explaining how to:
* Map TimeInterval with start or end to Period.
* Describe that the extension should be used otherwise.

And then apply this text to all occurrences of TimeInterval.

Reply:
Let's discuss thison Monday on how to best do this.

Reply:
This is handled through the branch/PR for zib-TimerInterval